pkg/importer/http-datasource.go

```diff
@@ -115,17 +118,25 @@ func (hs *HTTPDataSource) Info() (ProcessingPhase, error) {
 		klog.Errorf("Error creating readers: %v", err)
 		return ProcessingPhaseError, err
 	}
-	// The readers now contain all the information needed to determine if we can stream directly or if we need scratch space to download
-	// the file to, before converting.
-	if !hs.readers.Archived && !hs.customCA && !hs.brokenForQemuImg && hs.readers.Convert {
-		// We can pass straight to conversion from the endpoint. No scratch required.
-		hs.url = hs.endpoint
+	if hs.brokenForQemuImg {
+		return ProcessingPhaseTransferScratch, nil
+	}
+	hs.url = hs.endpoint
+	if !hs.readers.Archived && hs.customCA == "" && hs.readers.Convert {
```

Comment:
So this is not really a problem, but I think we can remove this check. With this check in place we will use qemu-img directly instead of nbdkit. If we remove this check we will use nbdkit for this particular case as well.